server/src/app.ts

```diff
@@ -16,7 +16,7 @@ import recommendedRecipes from "./routes/getRecommendedRecipes";
 const app = express();
 
 app.use(bodyParser.json());
-app.use(cors({credentials: true, origin: 'http://localhost:3000'}));
+app.use(cors({credentials: true, origin: 'https://foodease-client-snaanortey.vercel.app'}));
```

Comment:
we should store this value in an environment variable and use it here instead of being hard-coded.
When running locally, it will be `http://localhost` in your env file locally, however in vercel environment, it will be the vercel live URL 

```suggestion
app.use(cors({credentials: true, origin: 
```

Reply:
👍🏾👐🏾